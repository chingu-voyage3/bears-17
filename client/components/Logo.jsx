import React from 'react';

import logo from '../images/logo.svg';

const Logo = () => (
  <div className="logo">
    <a href="/" className="logo__link">
      <img src={logo} alt="Our app's name" />
    </a>
  </div>
);

export default Logo;
