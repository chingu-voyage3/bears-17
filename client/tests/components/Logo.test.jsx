import React from 'react';
import renderer from 'react-test-renderer';

import Logo from 'Components/Logo';

describe('Logo', () => {
  it('should match the snapshot', () => {
    const logo = renderer.create(<Logo />);

    expect(logo.toJSON()).toMatchSnapshot();
  });
});
