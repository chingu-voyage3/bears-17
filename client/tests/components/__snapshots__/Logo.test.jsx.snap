// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should match the snapshot 1`] = `
<div
  className="logo"
>
  <a
    className="logo__link"
    href="/"
  >
    <img
      alt="Our app's name"
      src="test-file"
    />
  </a>
</div>
`;
